lib/matplotlib/quiver.py

```diff
@@ -239,6 +174,52 @@ def __init__(self, Q, X, Y, U, label,
                  *, angle=0, coordinates='axes', color=None, labelsep=0.1,
                  labelpos='N', labelcolor=None, fontproperties=None,
                  **kw):
+        """
+        Add a key to a quiver plot.
+
+        Parameters
+        ----------
+        Q : `Quiver`
```

Comment:
`Quiver` is not linked because `matplotlib.quiver` is not listed in the modules section. Not sure if this is intentionally.

Anyway, the actual definition of `Quiver` is quite irrelevant. Most users will just do `q = plt.quiver()` and then `plt.quiverkey(q)`. And that's what the docs should tell them, e.g.

~~~
Q : `Quiver`
    A `Quiver` object as returned by a call to ``quiver()``.
~~~
I don't know if we want `quiver()` here as code or link to `pyplot.quiver` or Axes.quiver`.

Reply:
added quiver.py to modules section and edited docstring accordingly